components/combined-content-grid/Component.jsx

```diff
@@ -58,12 +58,14 @@ export default function FeaturedContent({
         <a
           href={card.liveUrl}
           aria-labelledby={`aria-announcement-${i}`}
-          className="su-group su-transition dark:su-text-digital-blue-vivid su-text-digital-blue su-flex su-flex-nowrap su-gap-2 su-items-center su-leading-[125%] su-text-16 su-font-semibold su-no-underline before:su-size-full before:su-top-0 before:su-left-0 before:su-absolute [&>svg]:su-size-16"
+          className="su-group su-transition dark:su-text-digital-blue-vivid su-text-digital-blue su-flex su-flex-nowrap su-gap-02em su-items-center su-leading-[125%] su-text-16 su-font-semibold su-no-underline su-stretched-link"
```

Comment:
I also use the `su-stretched-link` class here to replace `before:su-size-full before:su-top-0 before:su-left-0 before:su-absolute`. I believe the goal is to make the whole announcement teaser clickable and this class achieve the same results.